packages/terra-framework-docs/src/terra-dev-site/doc/compact-interactive-list/Examples/rowsData.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { IconDocuments, IconFeaturedOutline, IconImage } from 'terra-icon';
+
+// Source rows data for tests
+const rows = [
+  {
+    id: 'row_1',
+    cells: [
+      { content: <IconDocuments height="1.5em" width="1.5em" /> },
```

Comment:
The icons in this file are all meaningful, so they should make use of the `a11yLabel` prop.

Reply:
Added in https://github.com/cerner/terra-framework/pull/1869/commits/a60c1c52389d252309f3aeaed4b1de21ace9f470